flatfs.go

```diff
@@ -432,7 +432,7 @@ func (fs *Datastore) walk(path string, reschan chan query.Result) error {
 
 		key, ok := fs.decode(fn)
 		if !ok {
-			log.Warning("failed to decode entry in flatfs")
+			log.Warningf("failed to decode flatfs entry: %s", key)
 			continue
```

Comment:
`fn` should be used instead of `key` for the warning.  If there was an error `key` may be empty.

Reply:
D'oh -- I forgot to commit and push that last night after testing it with Gx :) Thanks.